meetings/2023-12-28.md

```diff
@@ -0,0 +1,96 @@
+# Node.js  Web Server Framework team Meeting 2023-18-12
+
+## Links
+
+* **Recording**:  
+* **GitHub Issue**: $GITHUB_ISSUE$
+
+## Present
+
+* Wes Todd - @wesleytodd 
+* Rafael Gonzaga - @RafaelGSS
+* Jean Burellier - @sheplu
+* Ethan Arrowood - @Ethan-Arrowood
+* Marco ippolito - @marco-ippolito
+* Paolo Insogna - @ShogunPanda
+
+## Agenda
+
+* Are we satisfied with continuing to be a “team” and running fairly loose?
+* What are our top priorities for 2024?
+* Milo
+* WinterCG Socket API Spec
+    * Spec: https://github.com/wintercg/proposal-sockets-api
+    * Node.js implementation: https://github.com/Ethan-Arrowood/socket
+    * Will be in draft mode until WinterCG becomes apart of an actual standards body (ECMA is under current proposal): https://github.com/wintercg/admin/issues/58
+    * Create `nodejs/socket` repository for Node.js implementation of Cloudflare's Socket API · Issue #826
+
+## Announcements
+
+N/A
+
+### nodejs/web-server-framework
+
+- Jean: Global presentation on the history + discussion during NodeconfEU
+    - WG started to focus on Http server, not other part of the stack
+    - Paolo: also we discussed other topics like the full http stack (dns, network, …)
+- Marco + Paolo: Maybe start working on the new socket API (presented by james during NodeConfEU)
+- Wes: global reflection about the name of the Working group. Do we want to rename to englobe a lot more than just HTTP ?
+- Ethan: Group started before Undici and some other projects. Primitives needed (and need) to be fixed. Depending on people we can decide on the scope.
+    - Better parser
+    - Better low level solution to build on top
+- Paolo: most of the groups overlap (between net, http…)
+- Wes: Team or WG next year? What would be the difference, different spaces ?
+- Ethan: Large scope and we should see what we want to be responsible for. Just including HTTP (http1, http2, quic, http3…) is already a really large scope
+- Marco: To not talk about the full ecosystem, do we want to focus on “only” the server part ?
+- Paolo: WG or team does not really matter. If we can define the interfaces, then all groups can work in parallel
+    - Marco: we should restrict the scope
+    - Wes: Parallel but with a priority list
+- James: What are the baseline problems that need to be addressed where the current core is falling short?
```

Comment:
He's missing on the "Present" list

Reply:
@jsumners

Reply:
I was trying to be an observer. But feel free to put me in the list.

Reply:
I didn't ping folks in my original comment, and turns out that was the right thing to do! Sorry for the ping, observe away!! But also I think I can speak for everyone that you are a great addition and your comments in the chat were helpful and additive!

Reply:
Don't worry. I'm not offended by being pinged. Just don't assume I will be present always or even really join the group. I might, but it's more likely I won't. 

Thank you for the appreciative words.